doc/tutorials/vcat_configuration.rst

```diff
@@ -0,0 +1,89 @@
+.. _vcat_configuration:
+
+Enable vCAT Configuration
+#########################
+
+vCAT is built on top of RDT, so to use vCAT we must first enable RDT.
+For details on enabling RDT configuration on ACRN, see :ref:`rdt_configuration`.
+For details on ACRN vCAT high-level design, see :ref:`hv_vcat`.
+
+The vCAT feature is disabled by default in ACRN. You can enable vCAT via the UI,
+the steps listed below serve as an FYI to show how those settings are translated
+into XML in the scenario file:
+
+#. Configure system level features:
+
+   - Edit :option:`hv.FEATURES.RDT.RDT_ENABLED` to `y` to enable RDT
+
+   - Edit :option:`hv.FEATURES.RDT.CDP_ENABLED` to `n` to disable CDP.
+     Currently vCAT requires CDP to be disabled.
+
+   - Edit :option:`hv.FEATURES.RDT.VCAT_ENABLED` to `y` to enable vCAT
+
+     .. code-block:: xml
+        :emphasize-lines: 3,4,5
+
+        <FEATURES>
+            <RDT>
+                <RDT_ENABLED>y</RDT_ENABLED>
+                <CDP_ENABLED>n</CDP_ENABLED>
+                <VCAT_ENABLED>y</VCAT_ENABLED>
+                <CLOS_MASK></CLOS_MASK>
+            </RDT>
+        </FEATURES>
+
+#. In each Guest VM configuration:
+
+   - Edit :option:`vm.guest_flags.guest_flag` and add ``GUEST_FLAG_VCAT_ENABLED``
+     to enable the vCAT feature on the VM.
+
+   - Edit :option:`vm.clos.vcpu_clos` to assign COS IDs to the VM.
+
+     If ``GUEST_FLAG_VCAT_ENABLED`` is not specified for a VM (abbreviated as RDT VM):
+     ``vcpu_clos`` is per CPU in a VM and it configures each CPU in a VM to a desired COS ID.
+     So the number of vcpu_closes is equal to the number of vCPUs assigned.
+
+     If ``GUEST_FLAG_VCAT_ENABLED`` is specified for a VM (abbreviated as vCAT VM):
```

Comment:
I'm not sure I understand the difference between what you call an RDT VM and vCAT VM, can you expand?

Reply:
RDT vm: GUEST_FLAG_VCAT_ENABLED is NOT specified, cat is not exposed to guest VM, acrn hypervisor allocate/init/manage the cat resource on behalf of the guest vm transparently.

vcat vm: GUEST_FLAG_VCAT_ENABLED is specified, cat is exposed to guest vm